src/function.py

```diff
@@ -495,14 +495,21 @@ def _package_log_payload(data):
     log_events = entry["logEvents"]
     log_messages = []
     lambda_request_id = None
+    trace_id = ""
 
     for log_event in log_events:
+        if LAMBDA_NR_MONITORING_PATTERN.match(log_event["message"]):
+            trace_id = _get_trace_id(log_event["message"])
+
         log_message = {
             "message": log_event["message"],
             "timestamp": log_event["timestamp"],
             "attributes": {"aws": {}},
         }
 
+        if trace_id:
+            log_message["traceId"] = trace_id
```

Comment:
Shouldn't this be `trace.id`?

Reply:
Do you mean former = `trace.id` vs latter = `traceId`. The example queries you posted are the same

Reply:
I was basing it off the struct tag [nr-lmbda-extension](https://github.com/newrelic/newrelic-lambda-extension/blob/01dc845bfecd63d9c4038e0f3cbe6a5b69be9daa/telemetry/payload.go#L102 )

Reply:
you are correct

Reply:
Whoops. Copy-pasted the same query twice. The second one should hve been `traceId`.